### PR TITLE
Update test results for Chrome 60 beta

### PR DESCRIPTION
**Pull request by mgol**
Tests run on Chrome beta 60.0.3112.32 on macOS.

### CODE REVIEW

File: data-es2016plus.js [hunk 1]
```diff
@@ -824,7 +824,7 @@ exports.tests = [
           firefox53: firefox.sharedmem,
           firefox55: true,
           chrome48: chrome.sharedmem,
-          chrome61: true,
```
Comment (ljharb) [inline data-es2016plus.js:827]: Im confused; why are there v61 results here before v60 is released?

Comment (mgol) [reply data-es2016plus.js:827]: Usually 2 Chrome pre-releases are shown in the table; people run tests on Canary before it reaches beta.

**Review by ljharb — APPROVED** [review]

File: data-es2016plus.js (hunk 1 above)
Comment (chicoxyzzy) [reply data-es2016plus.js:827]: shared memory was turned on by default in v8 6.0 just recently (when it became beta)

**Review by chicoxyzzy — APPROVED** [review]

**Comment by chicoxyzzy** [comment]
Thank you!

**Pull request merged**